src/Option_Command.php

```diff
@@ -436,6 +436,100 @@ public function update( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Gets the 'autoload' value for an option.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to get 'autoload' of.
+	 *
+	 * @subcommand get-autoload
+	 */
+	public function get_autoload( $args ) {
+		global $wpdb;
+
+		list( $option ) = $args;
+
+		$existing = $wpdb->get_row(
+			$wpdb->prepare(
+				"SELECT autoload FROM $wpdb->options WHERE option_name=%s",
+				$option
+			)
+		);
+		if ( ! $existing ) {
+			WP_CLI::error( "Could not get '{$option}' option. Does it exist?" );
+
+		}
+		WP_CLI::log( $existing->autoload );
+	}
+
+	/**
+	 * Sets the 'autoload' value for an option.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to set 'autoload' for.
+	 *
+	 * <autoload>
+	 * : Requires WP 4.2. Should this option be automatically loaded.
+	 * ---
+	 * options:
+	 *   - 'yes'
+	 *   - 'no'
```

Comment:
This means the API is:
```
wp option set-autoload my_option no
```
That seems to be forced by the DB value and is neither developer-speak (set autoload true) nor natural language (disable autoload).

I'm don't love that API, but I have trouble coming up with something better.

What do think about:
```
wp option autoload get my_option
wp option autoload enable my_option
wp option autoload disable my_option
```

Reply:
> I'm don't love that API, but I have trouble coming up with something better.
> 
> What do think about:

@schlessera Good points. I don't love your proposed approach either because it seems unnecessarily complex to nest the commands further.

I went with `wp option set-autoload my_option no` because `wp option add` accepts `--autoload=yes`/`--autoload=no`.